presto-docs/src/main/sphinx/release/release-0.116.rst

```diff
@@ -19,6 +19,16 @@ will be different from the semantics in 0.115 and before. When that comes,
 cast between JSON and VARCHAR in old scripts and views will produce unexpected
 result.
 
+Cluster Memory Manager Improvements
+-----------------------------------
+
+The cluster memory manager now has an OOM killer, similiar to the OOM killer
+in the Linux kernel. It can be enabled with the ``query.kill-on-out-of-memory``
```

Comment:
We should group these configs by subsystem:
* `query.oom-killer.enabled`
* `query.oom-killer.delay`